### PR TITLE
doc: fix 58710 issue docker install guide

### PR DESCRIPTION
**Pull request by generalova-kate**
### Details:
 - Fix for docker install guide

### Tickets:
 - 58710

### CODE REVIEW

**Review by ncybulsk — APPROVED** [review]
Verified

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 1]
```diff
@@ -27,19 +27,19 @@ Prebuilt images are available on:
 
 ## Build a Docker* Image
 
-You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci). The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
-You can also try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate the usage of OpenVINO™ Docker containers.
+You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci). 
+The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
+You can also try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate the usage of OpenVINO™ Docker containers. You can find device specific steps to configure OpenVINO Docker below.
```
Comment (andrew-zaytsev) [inline docs/install_guides/installing-openvino-docker-linux.md:32]: OpenVINO™ Docker containers --> Docker containers with Intel® Distribution of OpenVINO™ toolkit

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 2]
```diff
@@ -27,19 +27,19 @@ Prebuilt images are available on:
...
 - Kernel reports the same information for all containers as for native application, for example, CPU, memory information.
 - All instructions that are available to host process available for process in container, including, for example, AVX2, AVX512. No restrictions.
 - Docker\* does not use virtualization or emulation. The process in Docker* is just a regular Linux process, but it is isolated from external world on kernel level. Performance penalty is small.
 
-### <a name="building-for-cpu"></a>Build a Docker* Image for CPU
+### <a name="configuring-for-cpu"></a>Configure a Docker* Image for CPU
 
-You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci) for Intel® Distribution of OpenVINO™ toolkit. 
-The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
+You don't need to do specific steps to configure OpenVINO Dockerfile for CPU.
```
Comment (andrew-zaytsev) [inline docs/install_guides/installing-openvino-docker-linux.md:42]: 1) to configure OpenVINO Dockerfile for CPU --> to configure an Intel® Distribution of OpenVINO™ toolkit Dockerfile for CPU
2) What they actually need to do if they want to have Dockerfile for CPU?

File: docs/install_guides/installing-openvino-docker-linux.md [hunk 3]
```diff
@@ -261,17 +298,17 @@ docker run -it --rm --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp <image_name>
...
 ```sh
-docker run -it --rm --net=host -v /var/tmp:/var/tmp –ipc=host <image_name>
+docker run -it --rm --net=host -v /var/tmp:/var/tmp –-ipc=host <image_name>
 ```
 > **NOTES**:
 > 
-> - When building docker images, create a user in the docker file that has the same UID and GID as the user which runs hddldaemon on the host.
-> - Run the application in the docker with this user.
+> - When building Docker images, create a user in the Dockerfile that has the same UID(User Identifier) and GID(Group Identifier) as the user which runs hddldaemon on the host.
+> - Run the application in the Docker with this user.
```
Comment (andrew-zaytsev) [inline docs/install_guides/installing-openvino-docker-linux.md:311]: in the Docker --> in the Docker image

**Pull request merged**